ci/testgrid/config.yaml

```diff
@@ -125,6 +128,10 @@ dashboards:
     test_group_name: ci-knative-serving-api-coverage
     description: 'Conformance tests API coverage.'
     base_options: 'exclude-filter-by-regex=Overall$&group-by-directory=&expand-groups=&sort-by-name='
+  - name: performance
+    test_group_name: ci-knative-serving-performance-tool
+    description: 'Performance tests numbers'
```

Comment:
can we replace numbers with something more meaningful such as duration or latency or else?